docs/conf.py

```diff
@@ -86,7 +86,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-html_theme = "sphinx_rtd_theme"
+html_theme = "alabaster"
```

Comment:
Just curious on why the theme change? I think I do prefer "alabaster" to "rtd", but I think the paramount issue is being consistent throughout.